bin/utils/cli-config-utils.js

```diff
@@ -95,6 +95,38 @@ function success(text) {
   process.exit(0);
 }
 
+function print_migrations_list(completed, newMigrations) {
```

Comment:
I wouldn't say that config-utils is a good place for this particular set of logic, I would create a new file "migrationsLister". Also please use camelCase for function name ;)